modules/cactus-web/src/Breadcrumb/Breadcrumb.tsx

```diff
@@ -83,6 +86,26 @@ const BreadcrumbBase = (props: BreadcrumbProps): React.ReactElement => {
     [toggle, setFocus]
   )
 
+  const checkEllipsisVersion = React.useCallback(() => {
+    setEllipsisVersion(false)
+    const parentWidth = mainNavContainer.current?.parentElement?.getBoundingClientRect().width
+    const pivotWidth = pivotBreadcrumb.current?.getBoundingClientRect().width
+    if ((pivotWidth && parentWidth && pivotWidth >= parentWidth) || (isTiny && childrenCount > 2)) {
+      setEllipsisVersion(true)
+    }
```

Comment:
Couple of changes I recommend in this function:
```suggestion
    const parentWidth = Math.ceil(mainNavContainer.current?.parentElement?.getBoundingClientRect().width)
    const pivotWidth = Math.ceil(pivotBreadcrumb.current?.getBoundingClientRect().width)
    const ellipsisVersion = pivotWidth >= parentWidth || (isTiny && childrenCount > 2)
    setEllipsisVersion(ellipsisVersion)
```
The `Math.ceil` is to (hopefully) avoid rounding/truncation errors. Then instead of setting the state twice, I think you only really need to set it once.